presto-docs/src/main/sphinx/release/release-348.md

```diff
@@ -0,0 +1,86 @@
+# Release 348 (XX Dec 2020)
+
+## General Changes
+
+* Add support for `DISTINCT` clause in aggregations within correlated subqueries. ({issue}`5904`)
+* Improve query performance by reducing worker to worker communication overhead. ({issue}`6126`)
+* Reduce memory pressure and improve performance of queries that contain joins. ({issue}`6176`)
+* Fix ``EXPLAIN ANALYZE`` for certain queries that contain broadcast join. ({issue}`6115`)
+* Fix planning failures for queries that contain outer joins and aggregations using `FILTER (WHERE <condition>)` syntax. ({issue}`6141`)
+* Fix incorrect results when correlated subquery in join contains aggregation functions such as `array_agg` or `checksum`. ({issue}`6145`)
+* Fix duplicate call to event listener on query completion, if query failed early during preparation. ({issue}`6103`)
+* Support arbitrary query in `SHOW STATS`. ({issue}`3109`)
+* Output of `SHOW STATS` is changed ({issue}`3109`): 
```

Comment:
I am not sure if this is too detailed entry. Maybe we should just remove it?

Reply:
Yes, let's remove it.